e2e/e2etests/test_precompiles_distribute_through_contract.go

```diff
@@ -2,11 +2,14 @@ package e2etests
 
```

Comment:
Same for file renaming here

Reply:
Modified 4794a729e75dd587b32939b8ecd8963c56e2d00d